web/src/bundles/common.ts

```diff
@@ -11,6 +10,7 @@ import "source-code-pro/source-code-pro.css";
 import "@fontsource-variable/open-sans";
 import "../../third/bootstrap/css/bootstrap-alert.css";
 import "../../styles/alerts.css";
+import "../../styles/blueslip.css";
```

Comment:
With the logic for displaying the blueslip error being shifted to `blueslip.ts`, we only need the new `blueslip.css` to be added to `common.ts` bundle.